src/CosmosDB/CosmosDB/CosmosDB.csproj

```diff
@@ -6,7 +6,8 @@
 
   <Import Project="$(MSBuildThisFileDirectory)..\..\Az.props" />
   <ItemGroup>
-    <PackageReference Include="Microsoft.Azure.Management.CosmosDB" Version="3.3.0" />
+    <PackageReference Include="Azure.Security.KeyVault.Keys" Version="4.2.0" />
```

Comment:
```suggestion
    <PackageReference Include="Azure.Security.KeyVault.Keys" Version="4.2.0" />
```
is it necessary to add Azure.Security.KeyVault.Keys in CosmosDB.csproj?

Reply:
using KeyClient to create a keyvault and keys as part of TestHelper code. This keyvault will be used in tests.

Reply:
using KeyVault project reference as suggested.